src/jvm/main/org/jetbrains/kotlinx/lincheck/strategy/managed/ManagedStrategy.kt

```diff
@@ -848,6 +916,7 @@ abstract class ManagedStrategy(
         loopDetector.onThreadFinish(threadId)
         traceCollector?.onThreadFinish()
         unblockJoiningThreads(threadId)
+        tryAbortingUserThreads(threadId, null)
```

Comment:
Either make `blockingReason = null` as a default parameter value in `tryAbortingUserThreads`, or otherwise pass it by name: ` tryAbortingUserThreads(threadId, blockingReason = null)`